proto/reaction.proto

```diff
@@ -916,6 +916,7 @@ message Data {
     string value = 1;
     bytes bytes_value = 2;
     string url = 3;  // URL for data stored elsewhere.
+    float float_value = 6;
```

Comment:
I'm ok with adjusting the tag indices here. I think we should prefer to keep it cleaner/more readable until we move out of "alpha" testing.

Reply:
Fixed by c177aa9